samples/snippets/test_create_assessment.py

```diff
@@ -71,18 +72,32 @@ def recaptcha_site_key() -> str:
     )
 
 
+@pytest.mark.dependency()
 @pytest.mark.usefixtures("live_server")
 def test_create_assessment(
     capsys: CaptureFixture, recaptcha_site_key: str, browser: WebDriver
 ) -> None:
+    global ASSESSMENT_NAME
     token, action = get_token(recaptcha_site_key, browser)
     assess_token(recaptcha_site_key, token=token, action=action)
     out, _ = capsys.readouterr()
-    assert re.search("The reCAPTCHA score for this token is: ", out)
-    score = out.rsplit(":", maxsplit=1)[1].strip()
+    score = -1
+    for line in out.split("\n"):
+        if "The reCAPTCHA score for this token is" in line:
+            score = line.rsplit(":", maxsplit=1)[1].strip()
+        elif "Assessment name: " in line:
+            ASSESSMENT_NAME = line.rsplit(":", maxsplit=1)[1].strip()
+    assert score != -1 and ASSESSMENT_NAME != ""
```

Comment:
It might be easier to check for success by returning the entire `response` in the sample function so you can make asserts on the attributes here.

Reply:
Modified it in the update. Thanks